.circleci/config.yml

```diff
@@ -1081,7 +1081,7 @@ jobs:
       - run:
           name: Install dependencies
           command: |
-            pip install -ve .[docs]
+            pip install -ve .[docs] onnxruntime
```

Comment:
I accidentally forgot this in one run and got a reasonable message:
```
FAILED mne_bids_pipeline/tests/test_run.py::test_run[ERP_CORE_N170] - ImportError: Missing optional dependency. ICLabel requires either pytorch or onnxruntime. Use pip or conda to install one of them.
```
so I think the error message propagated to the end user is working!